advanced_html_css/animation/transforms.md

```diff
@@ -199,7 +199,7 @@ These are the additional transform function values to scale an element in a 3D s
 }
 ```
 
-See MDN's 3D cube in action with [`scaleZ`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scaleZ()) and [`scale3d`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scale3d()).
+See MDN's 3D cube in action with [`scaleZ`](<https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scaleZ()>) and [`scale3d`](<https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scale3d()>).
```

Comment:
```suggestion
See MDN's 3D cube in action with [`scaleZ`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scaleZ()) and [`scale3d`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scale3d()).
```

Reply:
Let me give it a go. Do I continue working on my branch, recommit/add/push to my fork or do I do something else to resolve these issues?

Reply:
Just keep working on the same branch - everything you push to it will be tracked here in this PR.

Reply:
Once you're done and there are no further issues flagged, just hit the "rerequest review" button in the top right
![image](https://github.com/TheOdinProject/curriculum/assets/122839503/aeab18b3-61c7-4156-8117-a47ec25be1fc)

And that will let me know it's ready to re-review (in case you push a commit but realised you missed something etc.).

Reply:
I disabled Prettier, ran `npm install` and `npm run fix:lesson -- path/to/file`, but the autofix did not work on `transforms.md`; oddly enough, it appears to have worked on `transforms.md`.